packages/bsky/src/services/moderation/index.ts

```diff
@@ -110,6 +112,12 @@ export class ModerationService {
       })
     }
 
+    if (reporters?.length) {
+      builder = builder.where((qb) => {
+        return qb.where('reportedByDid', 'in', reporters)
+      })
```

Comment:
I expect this can be simplified down to this:
```suggestion
      builder = builder.where('reportedByDid', 'in', reporters)
```

The way it's currently written would work fine too— the extra callback essentially just adds some parentheses around the expression in the SQL query.

Reply:
ah yes, I was in copy-paste mode, didn't pay much attention here. thanks!